### PR TITLE
#413 support overloading in graph construction

### PR DESCRIPTION
**Pull request by HDouss**
Resolving issue #413 
Add a puzzle to reenable tests after #437 is resolved.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#439](https://github.com/yegor256/jpeek/issues/439) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/6aa9dcbd-a1fc-48d0-8982-8d24f17f16fc); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/org/jpeek/graph/XmlGraph.java [hunk 1]
```diff
@@ -91,16 +88,40 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
...
                     caller.connections().add(callee);
                     callee.connections().add(caller);
                 }
             }
         }
         return new ListOf<>(byxml.values());
     }
+
+    /**
+     * Serialize method call to a string.
```
Comment (paulodamaso) [inline src/main/java/org/jpeek/graph/XmlGraph.java:106]: @HDouss I think that these two private methods could be extracted into two new classes

Comment (HDouss) [reply src/main/java/org/jpeek/graph/XmlGraph.java:106]: @paulodamaso I left a new puzzle for that.

**Comment by HDouss** [comment]
@paulodamaso ping

**Comment by paulodamaso** [comment]
@HDouss Thanks you!

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 28 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request merged**